src/app/components/play-round/play-round.component.ts

```diff
@@ -113,4 +118,11 @@ export class PlayRoundComponent implements OnInit {
     // Normalize scale
     return ((scale - 1) * 0.2) + 1;
   }
+
+  showTip(marker: RoundMarker) {
+    if (marker.isNest && !marker.canVisit) {
+      marker.tip = 'Gather Pollen to deposit them in your nest';
+    }
+  }
```

Comment:
I'm not a fan of functions that mutate their arguments. Could we have something like getTip(marker) that returns a string of the right tip?